cypress/integration/collection/ccLicense.spec.js

```diff
@@ -0,0 +1,29 @@
+import { PUBLISHED_ITEMS } from '../../fixtures/items';
+import { buildCollectionRoute } from '../../../src/config/routes';
+import { COLLECTION_LOADING_TIME } from '../../support/constants';
+import { SUMMARY_CC_LICENSE_CONTAINER_ID } from '../../support/selectors';
+import { buildPublicAndPrivateEnvironments } from '../../fixtures/environment';
+
+describe('CC License in Summary', () => {
+  buildPublicAndPrivateEnvironments().forEach((environment) => {
+    it(`Display item's cc license for ${environment.currentMember.name}`, () => {
+      cy.setUpApi(environment);
+
+      const item = PUBLISHED_ITEMS[0];
+      cy.visit(buildCollectionRoute(item.id));
+      cy.wait(COLLECTION_LOADING_TIME);
+
+      cy.get(`#${SUMMARY_CC_LICENSE_CONTAINER_ID}`).should('be.exist');
```

Comment:
Would it be possible to verify if the CC license is correct? (between the 2 possible options)

Reply:
The displayed element does not contain the value of the option, so it's hard to check that.